### PR TITLE
Don't provide access to Secrets in default RBAC

### PR DESCRIPTION
**Pull request by jarrpa**
Related: #164

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by jarrpa** [comment]
@msau42 PTAL

File: deploy/kubernetes/rbac.yaml [hunk 1]
```diff
@@ -21,9 +21,10 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: external-provisioner-runner
 rules:
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get", "list"]
+  # Some provisioners may need to uncomment the following rule.
```
Comment (msau42) [inline deploy/kubernetes/rbac.yaml]: Maybe something like "The following rule should be uncommented for plugins that require secrets for provisioning"

Comment (jarrpa) [reply deploy/kubernetes/rbac.yaml]: Comment updated.

**Pull request merged**